Library/Models/SurveyChannel.cs

```diff
@@ -0,0 +1,31 @@
+﻿//    This file is part of Nfield.SDK.
+//
+//    Nfield.SDK is free software: you can redistribute it and/or modify
+//    it under the terms of the GNU Lesser General Public License as published by
+//    the Free Software Foundation, either version 3 of the License, or
+//    (at your option) any later version.
+//
+//    Nfield.SDK is distributed in the hope that it will be useful,
+//    but WITHOUT ANY WARRANTY; without even the implied warranty of
+//    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+//    GNU Lesser General Public License for more details.
+//
+//    You should have received a copy of the GNU Lesser General Public License
+//    along with Nfield.SDK.  If not, see <http://www.gnu.org/licenses/>.
+
+using System;
+using Newtonsoft.Json;
+
+namespace Nfield.Models
+{
+    /// <summary>
+    /// Survey channel
+    /// </summary>
+    public enum SurveyChannel
```

Comment:
Incredible that this did not exist yet.. I guess because it was encoded in the survey type?

Reply:
The SurveyType enum has this types: Basic, Advanced, EuroBarometer, OnlineBasic, EuroBarometerAdvanced and Cati. I guess it's not the same, I took glu-api as reference, so I copied this enum from there

Reply:
Yeah, so CAPI has 4 survey types, online and CATI both have only one. This enum is more clear for sure.